variables.tf

```
@@ -398,6 +398,12 @@ variable "cpu_utilization_low_statistic" {
   description = "The statistic to apply to the alarm's associated metric. Either of the following is supported: `SampleCount`, `Average`, `Sum`, `Minimum`, `Maximum`"
 }
 
+variable "desired_capacity" {
+  type        = number
+  description = "The number of Amazon EC2 instances that should be running in the group, if not set will use `min_size` as value"
+  default     = 0
```

A desired count of 0 is a bit unnerving to me. A safer option is to set it to `null`.

```suggestion
  default     = null
```